scripts/taxonomy/OpenContextClassifierInput.py

```diff
@@ -1,4 +1,4 @@
-from ClassifierInput import ClassifierInput
+from scripts.taxonomy.ClassifierInput import ClassifierInput
 
 
 class OpenContextClassifierInput(ClassifierInput):
```

Comment:
I recommend this move to a new `isamples_metadata/taxonomy` directory.  This looks to be part of the core model infrastructure as opposed to a script that exercises it.  The `scripts` folder should really just be a collection of cmd-line stuff, while the core functionality should be in one the main folders under `src`.